pkg/games/playground/src/Playground.ts

```diff
@@ -157,6 +156,32 @@ export class Playground extends Game<PlaygroundProps> {
       id: "spaceship", renderer: this.props.renderer, networked: false,
       components: {
         position: new Position(100, 300),
+        interactive: new Interactive({
+          active: true,
+          bounds: { x: 50, y: 240, w: 100, h: 120 },
+          onPress: (e: Entity<EntityProps>, g: Game<GameProps>) => {
+            // const r = e.props.components.renderable as Character;
+            // r.currentAnimation.tint = 0x0000cc;
+
+            const spaceshipController = e.props.components.controller as Controller;
+            spaceshipController.active = true;
+
+            const skelly = g.props.entities["skelly1"] as Entity<EntityProps>;
+            const skellyController = skelly.props.components.controller as Controller;
+            skellyController.active = false;
+
+            g.props.renderer.trackCamera(e.props.components.renderable as Renderable<RenderableProps>);
```

Comment:
this logic for changing control to the spaceship is crude. needs a refactor once I find a good abstraction